libethereum/TransactionQueue.cpp

```diff
@@ -35,8 +35,8 @@ const size_t c_maxVerificationQueueSize = 8192;
 
 TransactionQueue::TransactionQueue(unsigned _limit, unsigned _futureLimit):
 	m_current(PriorityCompare { *this }),
-	m_limit(_limit),
-	m_futureLimit(_futureLimit)
+	m_limit(_limit * 10),
```

Comment:
that is a very unpredictable. A constructor should  set those numbers directly as expected.